lib/howitzer.rb

```diff
@@ -36,14 +36,25 @@ def mailgun_idle_timeout
   #  app_uri.site
   # @example returns url without auth
   #  app_uri.origin
+  # @example returns url for custom host(should be specified in config)
+  #  #config
+  #  example_app_host: 'example.com'
+  #  app_uri(:example).site
 
-  def self.app_uri
+  def self.app_uri(name = nil)
+    prefix = "#{name}_" if name.present?
     ::Addressable::URI.new(
-      user: Howitzer.app_base_auth_login,
-      password: Howitzer.app_base_auth_pass,
-      host: Howitzer.app_host,
-      scheme: Howitzer.app_protocol || 'http'
+      user: Howitzer.public_send("#{prefix}app_base_auth_login"),
+      password: Howitzer.public_send("#{prefix}app_base_auth_pass"),
+      host: Howitzer.public_send("#{prefix}app_host"),
+      scheme: Howitzer.public_send("#{prefix}app_protocol") || 'http'
     )
+  rescue NoMethodError
```

Comment:
Looks like settings are validating in SexySettings automatically. Do we really need the message?

Reply:
If setting is not present it throws exception like this:
```
undefined method `howitzer_app_base_auth_login' for Howitzer:Module (NoMethodError)
```
With this rescue message becomes more clear:
```
config is missing for custom host howitzer, please add to config:
howitzer_app_base_auth_login
howitzer_app_base_auth_pass
howitzer_app_host
howitzer_app_protocol (NoMethodError)
```